packages/blockchain-wallet-v4-frontend/src/data/session/slice.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { dissoc, mergeRight } from 'ramda'
+
+import { SessionStateType } from './types'
+
+const initialState: SessionStateType = {}
+
+const sessionSlice = createSlice({
+  initialState,
+  name: 'session',
+  reducers: {
+    removeSession: (state, action: PayloadAction<string>) => {
+      state = dissoc(action.payload, state.sessions)
+    },
+    saveSession: (state, action: PayloadAction<{ [key: string]: string }>) => {
+      state = mergeRight(state, action.payload)
```

Comment:
Same thing here, but looking into it, you don't actually know what the key/value is going to be so this seems right.